rotkehlchen/tests/api/test_history_base_entry.py

```diff
@@ -596,43 +592,3 @@ def mock_check_asset_existence(identifier: str, query_packaged_db: bool = True)
         'Could not deserialize one or more history event(s). '
         'Try redecoding the event(s) or check the logs for more details.',
     ]
-
-
-@pytest.mark.parametrize('number_of_eth_accounts', [0])
```

Comment:
we will need an endpoint like this I think. Don't think it's a good idea to remove it. Unless you have a way to satisfy the requirements I explained in the other comments

Reply:
Added this test back in, but modified it to use the new exchange specific endpoint.